config.json

```
@@ -24,7 +24,7 @@
       "test/%{snake_slug}_test.gleam"
     ],
     "example": [
-      "example/%{snake_slug}.gleam"
+      ".meta/%{snake_slug}.gleam"
```

As the example file is always the same (`example.gleam`), this should be:

```suggestion
      ".meta/example.gleam"
```